modules/sdk/gradle-plugins-node/src/main/java/com/liferay/gradle/plugins/node/NodeExtension.java

```diff
@@ -311,6 +358,73 @@ public void setYarnVersion(Object yarnVersion) {
 		_yarnVersion = yarnVersion;
 	}
 
+	private Optional<NodeInfo> _getNodeInfos(Path downloadPath)
```

Comment:
We should rename this method to `_getNodeInfoOptional` to reflect what is being returned.

Reply:
Or can we also inline this method?